.gitignore

```diff
@@ -1 +1,2 @@
 vendor
+.idea
```

Comment:
Please remove this line because this is IDE(PHPStorm) related and not a general rule and please preserve the empty new line at the end of the file.